lib/cogctl/actions/groups/add.ex

```diff
@@ -25,13 +25,13 @@ defmodule Cogctl.Actions.Groups.Add do
   end
 
   defp do_add(_endpoint, _, :undefined) do
-    display_error("Email address is required")
+    display_error("User is required")
```

Comment:
Would it be worthwhile to say something like "Cog username is required" instead, so there's no ambiguity (since you can be uniquely identified by your Cog username, your email address, or your chat handle)?